app/models/data_point.rb

```diff
@@ -61,12 +61,11 @@ def same_value_as?(value)
     end
     series = self.series
 
-    auto_quarantine_toggle = FeatureToggle.where(universe: self.universe, name: 'auto_quarantine').first || FeatureToggle.new(status: true)
-    if auto_quarantine_toggle.status && (Date.today - 2.years > self.date) && !series.quarantined && !series.restricted
-      series.update! quarantined: true
+    auto_quarantine = FeatureToggle.is_set('auto_quarantine', universe, true)
+    if auto_quarantine && (Date.today - 2.years > self.date) && !series.quarantined? && !series.restricted?
```

Comment:
Since we're in here, we should document what causes a series to be potentially quarantined (in the Udaman help).

Reply:
OK.